.changeset/smooth-countries-work.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/context': minor
```

Comment:
Just clarifying for myself that we're not updating `@lit-labs/context` anymore.

Reply:
Correct. Latest `@lit-labs/context` is just a re-export of `@lit/context@1`.